Microsoft.Toolkit.Uwp.UI.Controls/Menu/Menu.cs

```diff
@@ -57,6 +57,25 @@ public Orientation Orientation
                 new PropertyMetadata(Orientation.Horizontal, OrientationPropertyChanged));
 
         /// <summary>
+        /// Gets or sets the menu flyout placement
+        /// </summary>
+        public MenuFlyoutPlacement FlyoutPlacement
```

Comment:
What if the menu is at the middle of the screen !! I added it so it should be more fixable for user to choose whatever he wants and with "Auto" option we are supporting the default option.